.ruby-version

```diff
@@ -0,0 +1 @@
+2.0.0-p353
```

Comment:
Why don't you remove the file, add a git commit to remove it.  Open up `.gitignore` and add the file to it:

```
.ruby-version
```

Then you can specify it again locally:

```
rbenv local 2.0.0-p353
```

Then commit and push again.  I think that should do it.